modules/navier_stokes/doc/content/modules/navier_stokes/index.md

```diff
@@ -69,4 +69,30 @@ future these kernel sets will be consolidated into a single set.
+| Two-phase          | Mixture model             | Yes        | Yes                                                         | Yes                                                          |                        |
+|                    | Eulerian-Eulerian         |            |                                                             | Yes                                                          |                        |
+| Porous Flow        |       --                  | Yes        | Yes                                                         | Yes                                                          |                        |
+| Compressibility    | Incompressible            | Yes        | Yes                                                         | Yes                                                          | Yes                    |
+|                    | Weakly compressible       |            | Yes                                                         | Yes                                                          |                        |
+| Coupling           | Domain overlapping (SAM)  |            | Yes                                                         | Yes                                                          |                        |
+|                    | Direct to THM             |            | in [PR #28528](https://github.com/idaholab/moose/pull/28528)|                                                              |                        |
+|                    | MultiApp w/ Subchannel    |            | Yes                                                         |                                                              |                        |
+| Scalar Transport   |       --                  |            | Yes                                                         | Yes                                                          | Yes                    |
+| Physics Syntax     | Flow                      |            | Yes                                                         |                                                              | Yes                    |
+|                    | Fluid heat transfer       |            | Yes                                                         |                                                              | Yes                    |
+|                    | Solid phase heat transfer |            | Yes                                                         |                                                              |                        |
+|                    | Two phase                 |            | Yes                                                         |                                                              | in [PR #29614](https://github.com/idaholab/moose/pull/29614)|
```

Comment:
```suggestion
|                    | Two phase                 |            | Yes                                                         |                                                              | in [PR #29614](https://github.com/idaholab/moose/pull/29614) |
```
same for the others, it does a little odd thing if you dont have that space

Reply:
Comments addressed